experimental/handoff-library/csharp_dotnetcore/Microsoft.Bot.Builder.HandoffBot/.deployment

```diff
@@ -0,0 +1,2 @@
+[config]
+command = deploy.cmd
```

Comment:
Is this file needed? None of our other samples have it.

Reply:
These files are needed for build and deployment from within Azure portal (online editor). I removed them from git, kept in my own enlistment for when I need this.